tmpl/file.go

```diff
@@ -16,13 +16,14 @@ type templateFileRenderer struct {
 type TemplateData struct {
 	// Environment is accessible as `.Environment` from any template executed by the renderer
 	Environment environment.Environment
+	Namespace   string
```

Comment:
Adding a comment on a exported identifier is a good Go practice, what do you think about adding one here ?